Update go.mod

limzoo-afk opened the pull request:
github.com/codahale/hdrhistogram: github.com/codahale/hdrhistogram@v1.0.0: parsing go.mod:
        module declares its path as: github.com/HdrHistogram/hdrhistogram-go
                but was required as: github.com/codahale/hdrhistogram

suyash commented:
@limzoo-afk removing the dependency completely is a bad idea. Please update all references with the new import path. Additionally, I thought the `go.sum` file should prevent these problems, but anyway, please make sure the `go.sum` file is also updated as you do this.

/cc @natoscott

limzoo-afk commented:
> @limzoo-afk removing the dependency completely is a bad idea. Please update all references with the new import path. Additionally, I thought the `go.sum` file should prevent these problems, but anyway, please make sure the `go.sum` file is also updated as you do this.
> 
> /cc @natoscott

In fact, I found that the dependency has changed from github.com/codahale/hdrhistogram to github.com/HdrHistogram/hdrhistogram-go. I changed the corresponding package to make sure it can be compiled, but I did not verify that the functions are all normal, because my code There is no direct reference to it. In order to prevent problems, I do not submit directly. But I hope you guys can update it
